include/nodes/internal/ConnectionStyle.hpp

```diff
@@ -14,25 +14,25 @@ class NODE_EDITOR_PUBLIC ConnectionStyle : public Style
 
   ConnectionStyle();
 
-  ConnectionStyle(QString jsonText);
+  ConnectionStyle(const QString& jsonText);
```

Comment:
please put the `const` after the type everywhere. That's how the code was formatted in this project